android/src/main/java/com/mybigday/rnmediameta/RNMediaMeta.java

```diff
@@ -78,7 +79,18 @@ private void putString(WritableMap map, String key, String value) {
     if (value != null) map.putString(key, value);
   }
 
-  private void getMetadata(String path, Promise promise) {
+  private void getMetadata(String pathOrJSONString, Promise promise) {
```

Comment:
Since the method is actually ```get(path, promise)``` would you be suggesting that a second method definition would be created? (i.e.  ```get(path, options, promise)```)
I mention this because I tried it and React Native was throwing an error as it won't accept a duplicate method declaration. Got an example of how you would write it?